ert_gui/plottery/plots/histogram.py

```diff
@@ -166,7 +166,8 @@ def _plotHistogram(axes, plot_config, data, label, bin_count, use_log_scale=Fals
         minimum -= 0.5
         maximum += 0.5
 
-    axes.set_xlim(minimum, maximum)
+    if not use_log_scale or minimum>0:
```

Comment:
The condition here should really never be `False`, we should always enter the `if` statement. I'm not completely sure, but if setting limits to auto is the same as not setting them in the first place, then I would prefer:

```
axes.set_xlim(minimum, maximum)

if use_log_scale and minimum <= 0:
    axes.set_xlim(auto=True)
```
As that is the special case.

But is it necessary at all? From looking at the changes in `_histogramLogBins`, it seems we're trying to incorporate the negative value, and hence setting the limit non the less makes sense

Reply:
Setting the negative limit(s) for `log_scale` axis will through an error or a warning at least, therefore I would then suggest:
```python
if use_log_scale and minimum <= 0:
    axes.set_xlim(auto=True)
else:
    axes.set_xlim(minimum, maximum)
```

Reply:
Ah, of course, there's a `log_scale` axis, then I completely agree. Agree with the proposition